src/Angor/Client/Pages/Wallet.razor

```diff
@@ -29,7 +29,14 @@
     <div class="card card-body">
         <div class="d-flex align-items-center justify-content-between">
             <div class="d-flex col-auto my-auto">
-                <span class="material-icons fs-1 opacity-10 btn-angor user-select-none">wallet</span>
+                <span class="me-1">
+                    <svg width="64px" height="64px" viewBox="0 0 24 24" fill="none">
```

Comment:
same question here

Reply:
I've directly used SVG icon codes in Blazor code, which has significantly improved the loading speed of the icons. Additionally, managing their size and color has become much easier. Other benefits include better scalability, resolution independence, and reduced dependency on external icon libraries. These factors combined have enhanced the overall performance and maintainability of the application.

Reply:
I suggest we keep it in a static class and reference the text it should be the same loading speed just wont bulk the page